packages/target-nodejs-sdk/demo/demo.js

```diff
@@ -55,25 +52,24 @@ function getOffers() {
     })
     .then(res => {
       console.log("Result: Success", JSON.stringify(res, null, 4));
+    })
+    .catch(err => {
+      console.log(err.message);
     });
 }
 
-const targetClientOptions = {
+client = TargetClient.create({
   client: "adobesummit2018",
   organizationId: "65453EA95A70434F0A495D34@AdobeOrg",
-  clientReadyCallback: getOffers
-};
-
-client = TargetClient.create(
-  Object.assign(
-    {},
-    targetClientOptions,
-    doLocalDecisioning
-      ? {
-          executionMode: "local",
-          artifactLocation:
-            "https://target-local-decisioning-test.s3.us-west-2.amazonaws.com/adobesummit2018/waters_test/rules.json"
-        }
-      : {}
-  )
-);
+  fetchApi: getFetchWithProxy(),
+  executionMode: "local",
+  // environment: "staging",
+  // cdnEnvironment: "staging",
+  clientReadyCallback: getOffers,
+  logger: {
+    // eslint-disable-next-line no-console
```

Comment:
I think you can set this exception at the top of the file just once for the entire file.